pkg/plugin/plugin.go

```diff
@@ -55,6 +56,10 @@ func newRemotePlugin(details *registry.ProviderDetails, alias string, env []stri
 	rpcClient, err := client.Client()
 	if err != nil {
 		client.Kill()
+		// give a more clear message to the user
+		if strings.Contains(err.Error(), "Incompatible API version") {
```

Comment:
Are you sure the error comes in `client.Client()` and not few lines below in `Dispense()`?

Reply:
yeah. tested it.